pkg/api/types.go

```diff
@@ -1595,6 +1595,45 @@ type Handler struct {
 	TCPSocket *TCPSocketAction
 }
 
+// DeleteExecAction describes a "run in container" action that will be invoked
+// inside of a container prior to sending the TERM signal to the container's
```

Comment:
Note that now that https://github.com/kubernetes/kubernetes/pull/45236 has merged, the PID namespace is shared by default. I don't know that saying it sends TERM to the container's entrypoint makes sense any more. WDYT?